### ISSUE TITLE
package request: multipass

### ISSUE THREAD

**Issue by random-user-00** [clearlinux/distribution#2371]
**Official package name:** multipass

**License (must be an OSI approved Open Source license):** GPLv3

**Download URL of latest release:** https://github.com/canonical/multipass/releases

**Latest release date (must be recent):** 4 June 2021

**Description:**
Multipass is a lightweight VM manager for Linux, Windows and macOS. It's designed for developers who want a fresh Ubuntu environment with a single command.

**Comment by lebensterben** [comment]
> It's designed for developers who want a fresh Ubuntu environment with a single command.

Just use Ubuntu....

**Comment by fenrus75** [comment]
I assume you'd want it to spin quick clear linux vms ?

**Comment by random-user-00** [comment]
@fenrus75  yes! I just want to spin up Linux VMs really quick. It doesn't matter if VM image is CL or Ubuntu.
If there is any similar tool available for CL, it would be enough for me too.

**Comment by random-user-00** [comment]
For my purposes I am using Vagrant with libvirt and happy with it. I don't know if anyone else requires this package. Ubuntu is also making it harder to get their packages without Snap store.
I don't mind closing this one.
